guides/source/active_record_basics.md

```diff
@@ -142,7 +144,7 @@ end
 This will create a `Product` model, mapped to a `products` table at the
 database. By doing this you'll also have the ability to map the columns of each
 row in that table with the attributes of the instances of your model. Suppose
-that the `products` table was created using an SQL statement like:
+that the `products` table was created using a MySQL statement like:
```

Comment:
this is a SQL statement, not a MySQL statement, so let's keep it as it was.

Reply:
No, it's not. As I've written before, **syntax is different**!
https://www.w3schools.com/sql/sql_autoincrement.asp

Reply:
Lol! You are explaining @rafaelfranca what is a SQL statement. xD

Reply:
I'm sorry if that looked rude. I'm just telling what's written on the site linked above. If that's incorrect, then I apologize and ask for a better and accurate documentation (and tutorial, since I think that newcomers who don't know rDBMS should be presented with the opportunity to learn it).

Reply:
Actually `auto_increment` is a MySQL extension, but regardless of whether using any extensions or not, this is still an SQL statement.

Reply:
It is not incorrect as MySQL statement, but a MySQL statement is a SQL statement specific for MySQL. I think using SQL statement is better in this case because we are talking about the language, so the server name should not matter

Reply:
Also, don't worry about being rude, like any human I can be wrong too 😃 so don't be afraid of correcting me

Reply:
Again, my concern is not a technicality. It's just if a person, who is not familiar with SQL (like I was before this), would try to google the statement could be confused.

Btw, if we just remove AUTO_INCREMENT, would this statement be viable? Is it necessary? Since tutorial is supposed to be about Rails, the fewer nuances of SQL mentioned (especially regarding different implementations of the language), the better.

Reply:
@rafaelfranca I think I slightly appreciate @loothunter1's concern for the differentiation. For newbies who might be encountering SQL statements for the first time, this forgivable error could hamper their understanding of SQL statements down the line. Also, do note that by default Rails uses SQLite3 which doesn't have the `auto_increment` extension and so if the learner takes the pain to inspect their generated SQL they could be thrown off on an error that actually isn't. Please consider these arguments and see if we can make any changes. Thank you for making our lives easier and more productive everyday :smile: